electrophysiology/models.py

```diff
@@ -28,9 +28,11 @@ class ExtracellularRecording(BaseExperimentalData):
     filter_info = models.CharField(max_length=255, null=True, blank=True,
                                    help_text="Details of hardware corner frequencies, filter type, order. TODO: make this more structured?")
     start_time = models.FloatField(null=True, blank=True,
-                                   help_text="in seconds relative to experiment start. TODO: not DateTimeField? / TimeDifference")
+                                   help_text="in seconds relative to experiment start.") 
+#								   TODO: not DateTimeField? / TimeDifference")
+# the idea is that the experiment has a single DateTime when it started, then all times are in seconds relative to this. TimeDifference could work also, but why not just float?
```

Comment:
We certainly want the timestamp fields in file.timestamps.npy to be in seconds relative to experiment onset. 

I guess this doesn't necessarily mean they need to be like that in the database. The idea was that having them as relative times also in the database would make it easier to search multiple files within the database, and to relate them to the timestamps in the npy files.

The case you mentioned: searching for all recordings on a particular day, could still be done using the DateTimeField in the Experiment structure?